lib/admin/isAdmin.ts

```diff
@@ -0,0 +1,23 @@
+/**
+ * Get admin addresses from environment variable
+ * @returns Array of admin addresses (lowercased and trimmed)
+ */
+export const getAdminAddresses = (): string[] => {
+  const adminAddresses = process.env.ADMIN_ADDRESSES;
+  if (!adminAddresses) return [];
+  return adminAddresses.split(",").map((addr) => addr.toLowerCase().trim());
+};
```

Comment:
Single Responsibility Principle
- actual: 2 functions defined in one file
- required: new function file for getAdminAddresses